docs/Solana/DataFeeds/using-data-feeds-off-chain.md

```diff
@@ -18,10 +21,21 @@ To get the full list of Chainlink Data Feeds on Solana, see the [Solana Feeds](/
 
 **Table of contents:**
 
-- [Install the required tools](#install-the-required-tools)
-- [Run the example program](#run-the-example-program)
+- [The Chainlink Data Feeds Store Program](#the-chainlink-data-feeds-store-program)
+- [Using the Solana Starter Kit](#using-the-solana-starter-kit)
```

Comment:
why was the order switched on both pages to be starter kit first and then 'add to existing project' second? To cater more to beginners? Because most projects that integrate our price feeds will be adding them to existing projects, and won't use the starter kits. The starter kit are for people playing around and wanting to see how chainlink works on solana.

Reply:
Restored the original order.